### PR TITLE
Synnono Meme Foods Recipe Expansion 

### PR DESCRIPTION
**Pull request by Synnono**
- Adds 26 new recipes (18 Microwave, 5 Oven, 3 Fryer) to the kitchen, including curries, dank (and sliceable) brownies, and bear meat recipes!

- Adds 2 new reagents: Space Spice, a condiment found in the kitchen flour closet and used in many of the new recipes, and Brownie Mix, a dry mix of flour, sugar and cocoa that serves as the base for brownie recipes and doesn't conflict with other baked recipes.

- Adds sprites for new foods, as well as new trash sprites for dumplings and brownies.

### CODE REVIEW

File: code/modules/food/recipes_microwave.dm [hunk 1]
```diff
@@ -724,3 +724,160 @@ I said no!
...
+//in some foods. Many require the space spice reagent/condiment
+//to reduce the risk of future recipe conflicts.
+/////////////////////////////////////////////////////////////
+
+
+/datum/recipe/redcurry
+	reagents = list("cream" = 5, "spacespice" = 2, "rice" = 5)
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/cutlet,
+		/obj/item/weapon/reagent_containers/food/snacks/cutlet,
```
Comment (skull132) [inline code/modules/food/recipes_microwave.dm]: Unclosed list (comma at the end of last object).

File: code/modules/food/recipes_microwave.dm [hunk 2]
```diff
@@ -724,3 +724,160 @@ I said no!
...
+	)
+	reagent_mix = RECIPE_REAGENT_REPLACE //Simplify end product
+	result = /obj/item/weapon/reagent_containers/food/snacks/redcurry
+
+/datum/recipe/greencurry
+	reagents = list("cream" = 5, "spacespice" = 2, "rice" = 5)
+	fruit = list("chili" = 1)
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/tofu,
+		/obj/item/weapon/reagent_containers/food/snacks/tofu,
```
Comment (skull132) [inline code/modules/food/recipes_microwave.dm]: Ditto.

File: code/modules/food/recipes_microwave.dm [hunk 3]
```diff
@@ -724,3 +724,160 @@ I said no!
...
+		/obj/item/weapon/reagent_containers/food/snacks/doughslice,
+		/obj/item/weapon/reagent_containers/food/snacks/rawcutlet
+	)
+	reagent_mix = RECIPE_REAGENT_REPLACE //Water used up in cooking
+	result = /obj/item/weapon/reagent_containers/food/snacks/meatbun
+
+/datum/recipe/custardbun
+	reagents = list("spacespice" = 1, "water" = 5, "egg" = 3)
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/doughslice,
```
Comment (skull132) [inline code/modules/food/recipes_microwave.dm]: Tritto.

File: code/modules/food/recipes_microwave.dm [hunk 4]
```diff
@@ -724,3 +724,160 @@ I said no!
...
+	reagent_mix = RECIPE_REAGENT_REPLACE //Simplify end product
+	result = /obj/item/weapon/reagent_containers/food/snacks/chickenmomo
+
+/datum/recipe/veggiemomo
+	reagents = list("spacespice" = 2, "water" = 5)
+	fruit = list("carrot" = 1, "cabbage" = 1)
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/doughslice,
+		/obj/item/weapon/reagent_containers/food/snacks/doughslice,
+		/obj/item/weapon/reagent_containers/food/snacks/doughslice,
```
Comment (skull132) [inline code/modules/food/recipes_microwave.dm]: Quatro?

File: code/modules/food/recipes_microwave.dm [hunk 5]
```diff
@@ -724,3 +724,160 @@ I said no!
...
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/egg
+	)
+	reagent_mix = RECIPE_REAGENT_REPLACE //Get that water outta here
+	result = /obj/item/weapon/reagent_containers/food/snacks/poachedegg
+
+/datum/recipe/honeytoast
+	reagents = list("honey" = 5)
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/breadslice,
```
Comment (skull132) [inline code/modules/food/recipes_microwave.dm]: _hnrg_ meme is hard to continue.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 6]
```diff
@@ -2579,6 +2579,331 @@
...
+	bitesize = 3
+
+/obj/item/weapon/reagent_containers/food/snacks/bearburger
+	name = "bearburger"
+	desc = "The solution to your unbearable hunger."
+	icon_state = "bearburger"
+	filling_color = "#5d5260"
+	center_of_mass = list("x"=15, "y"=11)
+
+/obj/item/weapon/reagent_containers/food/snacks/bearburger/New()
```
Comment (skull132) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Waiting for Lohikar to yell at you to use `Initialize()` in 3...

Comment (Lohikar) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Alas, Initialize() is strictly verboten on reagent_containers.

Comment (skull132) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: awww.

Comment (Synnono) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Soooo, that's a no, then?

Comment (Lohikar) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: It's a no, `Initialize()` does not work on subtypes of `reagent_containers`.

**Pull request merged**